src/Packages.res

```diff
@@ -23,21 +23,28 @@ type npmPackage = {
   description: string,
   repositoryHref: Js.Null.t<string>,
   npmHref: string,
+  searchScore: float,
+  maintenanceScore: float,
+  score: {"final": float, "detail": {"quality": float, "popularity": float, "maintenance": float}},
```

Comment:
Is there any reason why you prefer an object here? Otherwise I suggest to make it a record.

Reply:
I was just copying from the raw source type to test out an idea, but I didn't end up using it so I went ahead and removed this from the type.